Source/JavaScriptCore/runtime/JSCConfig.h

```diff
@@ -123,7 +123,11 @@ constexpr size_t alignmentOfJSCConfig = std::alignment_of<JSC::Config>::value;
 static_assert(WTF::offsetOfWTFConfigExtension + sizeof(JSC::Config) <= WTF::ConfigSizeToProtect);
 static_assert(roundUpToMultipleOf<alignmentOfJSCConfig>(WTF::offsetOfWTFConfigExtension) == WTF::offsetOfWTFConfigExtension);
 
-#define g_jscConfig (*bitwise_cast<JSC::Config*>(&g_wtfConfig.spaceForExtensions))
+// Workaround to localize bounds safety warnings to this file.
+// FIXME: Use real types to make materializing JSC::Config* bounds-safe and type-safe.
+inline Config* addressOfJSCConfig() { return bitwise_cast<Config*>(&g_wtfConfig.spaceForExtensions); }
```

Comment:
Can we remove some of the `WTF_ALLOW_UNSAFE_BUFFER_USAGE` statements in WebKit now? Several remain only to wrap `g_jscConfig` & `g_wtfConfig` iirc.

Reply:
Yes, I think we can. Why don't I add a test removal of one of them to this patch.

Reply:
OK, added.